builder/compose/deps.bzl

```diff
@@ -1,13 +1,6 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
 
 def deps():
-    http_file(
-        name = "org_jetbrains_compose_compiler",
-        urls = ["https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/compiler/compiler-hosted/1.0.0-rc3/compiler-hosted-1.0.0-rc3.jar"],
-        sha256 = "1e628a81ad501bc737ca4ea193b8c5072a4baa022b3152aacb928bf076af8226",
-        downloaded_file_path = "compiler-hosted-1.0.0-rc3.jar",
-    )
-
```

Comment:
We don't need to specify the compose compiler this way anymore! We just specify it as a regular dependency, seen in //library/maven/artifacts.bzl.